.github/workflows/check_tests.yml

```diff
@@ -82,6 +82,16 @@ jobs:
         run: |
           source ${HOME}/.cargo/env
           RUSTC_BOOTSTRAP=1 cargo test --release --features=runtime-benchmarks,try-runtime
+      - name: Run Integration Tests
+        env:
+          RUST_BACKTRACE: full
+          RUSTC_WRAPPER: sccache
+          SCCACHE_CACHE_SIZE: 2G
+          SCCACHE_DIR: /home/runner/.cache/sccache
+        run: |
+          source ${HOME}/.cargo/env
+          RUSTC_BOOTSTRAP=1 cargo test -p integration-tests --release --features=calamari --no-default-features
+          RUSTC_BOOTSTRAP=1 cargo test -p integration-tests --release --features=manta --no-default-features
```

Comment:
Don't think that's possible cause in xcm-test it uses 

```rust 
if #[feature = "calamari"] {
...
} else {
...
}
```

Reply:
hm, i guess that's the cost of having these in unified code then.
I guess we can live with that, maintainability wins out.

Reply:
I added part to here, just because there's a lint check running `SKIP_WASM_BUILD=1 cargo check --no-default-features`, `no-default-features` will cause compilation error without this part of code.
https://github.com/Manta-Network/Manta/blob/manta/.github/workflows/run_linters.yml#L97